src/menu/menu.c

```diff
@@ -23,6 +23,7 @@
 #define MENU_DIRECTORY          "/menu"
 #define MENU_SETTINGS_FILE      "config.ini"
 #define MENU_CUSTOM_FONT_FILE   "custom.font64"
+#define MENU_HISTORY_FILE       "history.ini"
```

Comment:
given this is menu related, and particular reason why you should not just use the current `config.ini` with a seperate header?

Reply:
At first I felt it was better to have them in their own file out of the way of the config.ini
that way I could load and save as needed.
also makes it easier to wipe in a single go.